src/main/java/org/postgresql/sql2/PGConnectionProperties.java

```diff
@@ -360,7 +362,12 @@
   /**
    *
    */
-  TIMEZONE(String.class, "", false);
+  TIMEZONE(String.class, "", false),
+
+  /**
+   * Allows specifying the {@link NioLoop}.
+   */
+  NIO_LOOP(String.class, "", false);
```

Comment:
This should be 
NIO_LOOP(NioLoop.class, null, false);

Reply:
Yes, happy to change.   Focus for me was to get something together to prove the concepts.  Will fix :)